common/common.scss

```
@@ -1,9 +1,29 @@
 .about__main-content {
   .--custom-group {
+    max-width: unset !important; // ovverides ridiculously specific core style
     margin-top: 3em;
-    @if $capitalize-names == "true" {
+    h3 {
+      a {
+        color: var(--primary);
+        &:hover {
+          color: var(--tertiary);
+        }
+      }
+      @if $capitalize-names == "true" {
+        &:first-letter {
```

"technically correct is the best correct" 😄 

```suggestion
        &::first-letter {
```